examples/phi3/phi3.py

```diff
@@ -0,0 +1,216 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import argparse
+import json
+import time
+from pathlib import Path
+
+import onnxruntime_genai as og
+
+from olive.workflows import run as olive_run
+
+# flake8: noqa: T201
+
+
+TARGETS = ["cpu", "gpu", "mobile", "web"]
```

Comment:
the targets don't match the actual options. needs to be "cuda" instead of "gpu"

Reply:
Good call! i didn't get a chance to check phi3 example code. I've addressed all comments.